README.md

```diff
@@ -301,9 +302,22 @@ service:
       exporters: [opencensus, zipkin]
 ```
 
-## Other Information
+### <a name="troubleshooting"></a>Troubleshooting
+By default, the OpenTelemetry Collector exposes Prometheus metrics and logs for
+monitoring and troubleshooting. When troubleshooting live issues it is 
+recommended to use `zpages` [extension](#config-extensions).
 
-### Extending the Collector
+The `zpages` extension provides live information about receivers and exporters.
+By default, `zpages` is available
+at `http://localhost:55679/debug/tracez`. Click on the links of the displayed
+operations to see information about each individual operation. Operations that
+encountered errors are reported on the right most column.
```

Comment:
Nice! Now I will know how to use it. It was always a mystery to me that I kept postponing to understand.